packages/editor-ui/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
To learn: This does not belong in `/public` anymore?

Reply:
Vite.js expects `index.html` to be in the project root and have references to build entry points. Everything in `public` will still be copied over though.